repair/repair.cc

```diff
@@ -742,6 +742,30 @@ future<> repair::shard_repair_task_impl::repair_range(const dht::token_range& ra
         co_return;
     }
     try {
```

Comment:
>  Tracing doesn't support
embedded sessions natively, so we initialize a new tracing session and

What about the span thing? b0f660331acf25158f0eab63112c625ba94deee2